src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateEvaluator.cs

```diff
@@ -425,8 +400,7 @@ public class PipelineTemplateEvaluator
         private TemplateContext CreateContext(
             DictionaryContextData contextData,
             IList<IFunctionInfo> expressionFunctions,
-            IEnumerable<KeyValuePair<String, Object>> expressionState = null,
-            bool setMissingContext = true)
+            IEnumerable<KeyValuePair<String, Object>> expressionState = null)
```

Comment:
i reverted changes from this file back to their original state before composite-actions-changes

switched to PipelineTemplateConverter instead to convert steps token into `Pipelines.Steps`